### DISCUSSION TITLE
Deleting branches after merge

### DISCUSSION THREAD

**Discussion by DavidBiddle** [govuk-forms/forms#40] [Polls]
Should we turn on automatic deletion of feature branches after pull requests are merged? We're generally doing this manually after every PR anyway, so it might make sense to automate it.

Branches in Git are cheap, so it doesn't hugely matter if they aren't deleted, but it's tidier if we delete them - e.g. if we look at [Github's branch view for the admin](https://github.com/alphagov/forms-admin/branches) we can see there are more merged branches still hanging around than there are active branches. It's easy to recreate a branch once it's been deleted, so there's no real downside to doing this.

**Comment by aliuk2012** [comment]
💯 agree, as a dev I dont want to have to remember to manually clear up branches after myself.

**Comment by DavidBiddle** [comment]
I think if nobody objects today, I'll do this tomorrow morning pre-standup

**Reply by DavidBiddle** [reply]
Done!
